mountpoint-s3-crt/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## Unreleased
 
+## v0.8.1 (September 12, 2024)
+
 * Update to latest CRT dependencies
```

Comment:
I should have added a change log entry here. I think we bump to v0.9.0 and incloude something like:

```
* Allow specifying a list of network interfaces to be used by an S3 client
```